components/supervisor/frontend/src/shared/loading-frame.ts

```diff
@@ -16,7 +16,7 @@ export function load(): Promise<{
         frame.src = startUrl.toString();
         frame.style.visibility = "visible";
         frame.className = "gitpod-frame loading";
-        document.body.appendChild(frame);
+        document.body.prepend(frame);
```

Comment:
This solves a bug in which in Safari, the loading screen would not be hidden when the IDE loaded. It wasn't present in VS Code, because we actually append the IDE itself lazily, but for Xterm and potentially other web IDEs we might have a static HTML structure ready.